pallets/nft/src/mock.rs

```diff
@@ -121,10 +122,37 @@ pub fn run_to_block(n: u32) {
 	}
 }
 
-pub fn events() -> Vec<Event> {
-	let evt = System::events().into_iter().map(|evt| evt.event).collect::<Vec<_>>();
+pub fn events_filter<T: 'static>() -> Vec<Event> {
```

Comment:
Even it's clear under your test examples, pls add some comments about the usage of these functions here so that ppl who just look at here can understand what are these and how they work together

Reply:
comment added